### PR TITLE
Remove Python 3.9 tests

### PR DESCRIPTION
**Pull request by brunoapimentel**
Dependabot PR's have been failing because of dependency resolution in Python 3.9, since 'pip-compile' is run on Python 3.10 env.

To solve this issue, we decided to drop Python 3.9 tests.

# Maintainers will complete the following section

- [ ] Commit messages are descriptive enough
- [ ] Code coverage from testing does not decrease and new code is covered
- [ ] New code has type annotations
- [ ] OpenAPI schema is updated (if applicable)
- [ ] DB schema change has corresponding DB migration (if applicable)
- [ ] README updated (if worker configuration changed, or if applicable)

### CODE REVIEW

**Comment by lkolacek** [comment]
PR looks good so far. Do we want to remove rest of Py3.9 leftovers in other files too? (README.md, docs/using_requests_locally.md, requirements-test.txt, setup.py)

**Comment by chmeliik** [comment]
Should we drop the dependabot commit or is it intentional?

**Comment by brunoapimentel** [comment]
> Should we drop the dependabot commit or is it intentional?

It was meant to test the changes, and I initially thought it could be added. But on a second thought, it might be clearer if we drop it and merge the individual dependabot PRs.

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #
-# This file is autogenerated by pip-compile with python 3.9
+# This file is autogenerated by pip-compile with python 3.10
```
[the remark this answers is not in the document]
Comment (brunoapimentel) [reply requirements.txt:2]: Yeah, the requirements-web.txt file had some more changes... requirements.txt was already correct.

**Review by chmeliik — APPROVED** [review]
LGTM :+1:

**Review by ejegrova — APPROVED** [review]
LGTM

**Pull request merged**